appveyor.yml

```diff
@@ -1,5 +1,5 @@
 environment:
-  nodejs_version: "6"
+  nodejs_version: "7"
```

Comment:
why 7 and not 8?

Reply:
8 caused the issues with WebDriverAgent privateheaders

Reply:
ok